pessimistic-transaction.md

```diff
@@ -121,7 +121,13 @@ TiDB supports the following two isolation levels in the pessimistic transaction
 
 Adding a pessimistic lock requires writing data into TiKV. The response of successfully adding a lock can only be returned to TiDB after commit and apply through Raft. Therefore, compared with optimistic transactions, the pessimistic transaction mode inevitably has higher latency.
 
-To reduce the overhead of locking, TiKV implements the pipelined locking process: when the data meets the requirements for locking, TiKV immediately notifies TiDB to execute subsequent requests and writes into the pessimistic lock asynchronously. This process reduces most latency and significantly improves the performance of pessimistic transactions. However, there is a low probability that the asynchronous write into the pessimistic lock might fail, resulting in the commit failure of the pessimistic transaction.
+To reduce the overhead of locking, TiKV implements the pipelined locking process: when the data meets the requirements for locking, TiKV immediately notifies TiDB to execute subsequent requests and writes into the pessimistic lock asynchronously. This process reduces most latency and significantly improves the performance of pessimistic transactions. However, when TiKV has network isolation or nodes downtime, the asynchronous write into the pessimistic lock might fail, resulting in the following effects:
+
+* Other transactions that modify the same data cannot be blocked. If the application logic relies on locking or lock waiting mechanisms, it affects the correctness of the application logic.
+
+* Though there is a low probability that the transaction commit fails, it does not affect the correctness of the transactions.
```

Comment:
The key point of this sentence is that **there is a low probability that the transaction commit fails**.

I'm not sure if we should change it to:
There is a low probability that the transaction commit fails, but it does not affect the correctness of the transactions.